src/openloco/ui/scrollview.cpp

```diff
@@ -2,11 +2,161 @@
 #include "../interop/interop.hpp"
 #include "../ui.h"
 #include "WindowManager.h"
+#include <cmath>
 
 using namespace openloco::interop;
 
 namespace openloco::ui::scrollview
 {
+    static constexpr uint16_t thumbSize = 10;
+    static constexpr uint16_t barWidth = 11;
+
+    // 0x004C87E1
+    void scroll_5_follow(WindowType _pressedWindowType, window_number _pressedWindowNumber, widget_index _pressedWidgetIndex, size_t _currentScrollOffset, int16_t ax)
+    {
+        auto window = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
+        if (window == nullptr)
+            return;
+
+        scroll_area_t& scrollArea = window->scroll_areas[_currentScrollOffset];
```

Comment:
At this point, this isn't `_currentScrollOffset` anymore. You've already divided it in the call.